src/Migrations.php

```diff
@@ -403,7 +403,7 @@ public function setAdapter()
      */
     public function getInput($command, array $arguments, array $options)
     {
-        $className = '\Migrations\Command\\' . $command;
+        $className = '\Migrations\Command\Phinx\\' . $command;
```

Comment:
Leading slash should not be necessary.

Reply:
needs because then infers `'` is a char but is not